CHANGELOG.md

```diff
@@ -26,16 +26,28 @@ Highlights
   `--cubical-compatible`, or if it is already making extensive use of
   indexed types.
 
+  Note that code that uses (only) `--without-K` can no longer be
+  imported from code that uses `--cubical`. Thus it may make sense to
+  replace `--without-K` with `--cubical-compatible` in library code,
+  if possible.
+
+  Note also that Agda tends to be quite a bit faster if `--without-K`
+  is used instead of `--cubical-compatible`.
+
+* Agda 2.6.3 seems to type-check one variant of the standard library
+  about [30%
+  faster](https://github.com/agda/agda/issues/6049#issuecomment-1329163727)
```

Comment:
Why this line break?  I think link texts shouldn't be submitted to line breaks...

Reply:
I want to avoid long lines in the source code, and I would hope that this kind of line break is semantically equivalent to a single space character.